src/main/resources/plugin.yml

```diff
@@ -1,6 +1,7 @@
 name: uCars
 main: com.useful.ucars.ucars
-version: 20
+version: 20.75
+api-version: 1.16
```

Comment:
The master branch of uCars should be 1.12 compatible as the primary version for now. 1.16 can be a separate branch, or can also be supported in addition to 1.12 but 1.12 must be compatible.